packages/login/src/vaadin-login-form.js

```diff
@@ -60,11 +59,9 @@ class LoginForm extends LoginMixin(ElementMixin(ThemableMixin(PolymerElement)))
       <vaadin-login-form-wrapper
         theme$="[[_theme]]"
         part="vaadin-login-native-form-wrapper"
-        action="{{action}}"
-        disabled="{{disabled}}"
-        error="{{error}}"
-        no-forgot-password="{{noForgotPassword}}"
-        i18n="{{i18n}}"
```

Comment:
No need to bind `action` and `disabled` because the form wrapper doesn't use both properties.
Changed others to use one-way data binding, as they aren't changed in the form wrapper.